src/ucp/core/ucp_ep.h

```diff
@@ -390,10 +390,11 @@ typedef struct {
  * Endpoint extension for control data path
  */
 typedef struct {
-    ucs_ptr_map_key_t             local_ep_id;   /* Local EP ID */
-    ucs_ptr_map_key_t             remote_ep_id;  /* Remote EP ID */
-    ucp_err_handler_cb_t          err_cb;        /* Error handler */
-    ucp_ep_close_proto_req_t      close_req;     /* Close protocol request */
+    ucp_rsc_index_t          cm_idx; /* CM index */
```

Comment:
unrelated change of moving cm_idx?

Reply:
why can't we take it from wireup_ep?

Reply:
> why can't we take it from wireup_ep?

`wireup_ep` doesn't exist at the time when a connection establishment completed

Reply:
so what do we have? only pure cm lane?

Reply:
> so what do we have? only pure cm lane?

yes